common/pom.xml

```diff
@@ -207,6 +207,25 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>jdk11+</id>
+      <activation>
+        <jdk>[11,)</jdk>
+      </activation>
+      <properties>
+        <java.version>11</java.version>
+      </properties>
+        <dependencies>
+          <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+          </dependency>
+          <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+          </dependency>
+        </dependencies>
+    </profile>
```

Comment:
If the code is not compatible/compilable by Java 8 anymore why do we need a specific profile for Java >=11. Is not better to always add `jabx-api` and `javax.annotation-api` instead of having it in a specific profile?